Composer/packages/client/src/components/Notifications/useSurveyNotification.ts

```diff
@@ -12,7 +12,7 @@ import { MachineInfo } from '../../recoilModel/types';
 import { LAST_SURVEY_KEY, SURVEY_URL_BASE, SURVEY_PARAMETERS } from '../../constants';
 import TelemetryClient from '../../telemetry/TelemetryClient';
 
-const buildUrl = (info: MachineInfo) => {
+export const buildUrl = (info: MachineInfo) => {
```

Comment:
We typically shouldn't export something just for testability. Was this exported just to write unit tests?

Reply:
Removed this.